test/test_metapackage.py

```diff
@@ -26,9 +26,19 @@
     'no_buildtool_depend_catkin': [InvalidMetapackage, 'No buildtool dependency on catkin', None],
     'no_cmake': [InvalidMetapackage, 'No CMakeLists.txt', None],
     'no_metapackage_tag': [InvalidMetapackage, 'No <metapackage/> tag in <export>', None],
+    'NonConformingName': [None, None, None],
     'valid_metapackage': [None, None, None]
 }
 
+test_expected_warnings = [
+    'Metapackage "invalid_depends" should not have other dependencies besides a '
```

Comment:
Since you wrapped everything else to fit in 80 columns: the `a ` exceed the line length and should be wrapped to the next line.